lms/static/js/student_account/views/AccessView.js

```diff
@@ -1,280 +1,283 @@
-var edx = edx || {};
-
-(function($, _, _s, Backbone, History) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'utility', 'underscore', 'underscore.string', 'backbone', 'js/student_account/models/LoginModel',
+            'js/student_account/models/PasswordResetModel', 'js/student_account/models/RegisterModel',
+            'js/student_account/views/LoginView', 'js/student_account/views/PasswordResetView',
+            'js/student_account/views/RegisterView', 'js/student_account/views/InstitutionLoginView',
+            'js/student_account/views/HintedLoginView', 'js/vendor/history'],
+        function($, utility, _, _s, Backbone, LoginModel, PasswordResetModel, RegisterModel, LoginView, PasswordResetView,
+        RegisterView, InstitutionLoginView, HintedLoginView) {
+
+        if (_.isUndefined(_s)) {
+            _s = _.str;
+        }
 
-    edx.student = edx.student || {};
-    edx.student.account = edx.student.account || {};
-
-    edx.student.account.AccessView = Backbone.View.extend({
-        el: '#login-and-registration-container',
-
-        tpl: '#access-tpl',
-
-        events: {
-            'click .form-toggle': 'toggleForm'
-        },
-
-        subview: {
-            login: {},
-            register: {},
-            passwordHelp: {},
-            institutionLogin: {},
-            hintedLogin: {}
-        },
-
-        nextUrl: '/dashboard',
+        var AccessView = Backbone.View.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the viewl, i.e.
```
return Backbone.View.extend({
```